### ISSUE TITLE
Dimension expansion

### ISSUE THREAD

**Issue by dervishson** [keras-team/keras#13284]
Hi,
I have a problem in converting my 1-d tensor to two dimensional one. 
input----> output_pre1
Output-----> [<tf.Tensor 'dense_886/BiasAdd:0' shape=(?, 1) dtype=float32>]
My input is 1-d tensor, i want it to convert to shape=(?,1,1).In implementation of keras.backend.expand_dims(output_pre1, axis=0 or 1), it returns an input with shape like this: <tf.Tensor 'ExpandDims_10:0' shape=(1, 1, ?, 1) dtype=float32>, however i want the shape to be (?,1,1). How can i solve this problem?
